site/public/js/submission-page.js

```diff
@@ -1,5 +1,10 @@
-function showActions(form_id) {
-    var form = $("#" + form_id);
-    form.css("display", "block");
-    form.find('.form-body').scrollTop(0);
+
+function openActionsPopup(popup_css, server_css, element_id) {
+  var elem_html = "<link rel=\"stylesheet\" type=\"text/css\" href=\"" + server_css + "\" />";
+  elem_html += "<link rel=\"stylesheet\" type=\"text/css\" href=\"" + popup_css + "\" />";
+  elem_html += document.getElementById(element_id).innerHTML;
```

Comment:
there should probably be some minimal padding around the left/right side of content as having it start at 0px is a bit ugly.